Formula/fastme.rb

```diff
@@ -0,0 +1,34 @@
+class Fastme < Formula
+  desc "Accurate and fast distance-based phylogeny inference program"
+  homepage "http://www.atgc-montpellier.fr/fastme/"
+  url "https://gite.lirmm.fr/atgc/FastME/raw/master/tarball/fastme-2.1.5.2.tar.gz"
+  sha256 "834e1f555b6057b23650cb214c1f2f899978d72d085fca863fb3f03f2bbf00fc"
+
+  option "with-openmp", "Enable parallel computations with OpenMP"
+
+  depends_on "gcc" if build.with? "openmp"
```

Comment:
Use a block rather than two distinct `if build.with? "openmp"` statements.

```ruby
  if build.with? "openmp"
    depends_on "gcc"
    fails_with :clang
  end
```

Reply:
Done.